src/site/content/en/blog/ads-case-study-stale-while-revalidate/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How Google improved ads performance with stale-while-revalidate
+subhead: |
+    Find out how loading third-party resources faster can increase revenue.
+date: 2020-03-04
+authors:
+  - thebengeu
+  - jimper
+description: |
+    This case study explains how Google increased key ads business metrics by
+    optimizing the performance of their third-party ads script with stale-while-revalidate.
+tags:
+  - post
+  - case-study
+  - performance
+  - fast
+  - ads
+---
+
+This case study highlights how improving the performance of third-party resources can boost business metrics. While a [previous study](/fast-ads-matter/#fast-ads-make-you-more-money) measured the cost of added ads latency, this study demonstrates the value of a real-world performance improvement:
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">0.5<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Revenue lift for publishers</p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">2<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Increase in early ad script loads</p>
+  </div>
+</div>
+
+Source: Google Internal Data, June to July 2019.
+
+## About
+
+The Google Publisher Tag (GPT) is the ad tagging script for Google Ad Manager that requests and renders display ads on the web. By implementing a simple `stale-while-revalidate` HTTP header for GPT, we were able to improve the speed and performance of Google display ads for our publisher partners. This same technique can be applied to many other scenarios where loading scripts as quickly as possible is more important than loading the freshest code.
```

Comment:
```suggestion
The [Google Publisher Tag (GPT)](https://support.google.com/admanager/answer/181073) is the ad tagging script for Google Ad Manager that requests and renders display ads on the web. By implementing a simple [`stale-while-revalidate`](/stale-while-revalidate/) HTTP header for GPT, the GPT team was able to improve the speed and performance of Google display ads for its publisher partners. This same technique can be applied to any other scenario where loading scripts as quickly as possible is more important than loading the freshest code.
```

Changes:

* Linking to explanation of GPT
* Linking to Jeff's stale-while-revalidate article
* Removing the use of "we" because it slightly implies that web.dev == GPT team (it may seem like a nitpick but just take my word that there are bigger content strategy motivations behind this)

That explanation of how this is generally applicable to web developers at the end is GREAT!